public/app/features/logs/components/panel/virtualization.ts

```diff
@@ -248,14 +255,28 @@ export const calculateFieldDimensions = (logs: LogListModel[], displayedFields:
   return dimensions;
 };
 
-export function hasUnderOrOverflow(element: HTMLDivElement, calculatedHeight?: number): number | null {
+// 2/3 of the viewport height
+export const TRUNCATION_LINE_COUNT = Math.round(window.innerHeight / getLineHeight() / 1.5);
```

Comment:
FYI you can change this parameter to test other line limits. The current limit are lines that are around 50% of the viewport height.